tests/proofs/listsum-spec.k

```diff
@@ -1,16 +1,18 @@
-requires "lemmas.md"
-
 module VERIFICATION
-  imports LEMMAS
+  imports MICHELSON
 endmodule
 
+
 module LISTSUM-SPEC
   imports VERIFICATION
 
-  rule <k>  { ITER .AnnotationList { DROP .AnnotationList ; PUSH .AnnotationList nat .AnnotationList 1 ; ADD .AnnotationList } } => .K </k>
-       <stack> [ list nat NatList:InternalList ] ; [ nat 0 ] => [ nat ?N:Int ] </stack>
-     requires isValue(list nat, NatList)
-     ensures  ?N ==Int size(NatList, 0)
+  claim <k> ITER .AnnotationList {
+              DROP .AnnotationList ;
+              PUSH .AnnotationList nat .AnnotationList 1 ;
+              ADD .AnnotationList
+            } => . </k>
+        <stack> [ list int NS:InternalList ] ; [ int M:Int ]
+             => [ int M +Int size(NS) ] </stack>
```

Comment:
It's clear that the original claim didn't describe a proper loop invariant, because the intermediate sum is not always `0`.
It makes me wonder if this test ever passed.
Thanks for the fresh eyes and the fix.